### PR TITLE
Criteria: sandbox update/clarification

### PR DESCRIPTION
**Pull request by dwelsch-esi**
Updated the assessment criterion in the Website section of criteria.md to remove the requirement to request an assessment for sandbox projects.

Made numerous miscellaneous updates to techdocs//README.md and docs/analysis/criteria.md.

### CODE REVIEW

File: docs/analysis/criteria.md [hunk 1]
```diff
@@ -148,9 +151,9 @@ We evaluate on the following:
 
 - Is project governance clearly documented?
 
-Examples:
+Example:
 
-- Any graduated CNCF project
+- <https://github.com/kubernetes/community> (see `governance.md`)
```
Comment (nate-double-u) [inline docs/analysis/criteria.md:156]: a nit: do we want to use Kubernetes as an example here? It's got quite a complex governing structure, we may be better off to use an average one, or link to TAG Contributor Strategy's resources (https://contribute.cncf.io/maintainers/governance/)

Comment (dwelsch-esi) [reply docs/analysis/criteria.md:156]:
```suggestion
- <https://github.com/envoyproxy/envoy/blob/main/GOVERNANCE.md>
```

File: analyses/0011-keda/keda-implementation.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ Here is a proposed outline for the tech doc Table of Contents:
   - [Events](https://keda.sh/docs/2.13/operate/events/)
   - [Firewall requirements](https://keda.sh/docs/2.13/operate/cluster/#firewall)
   - ...
-  - [FAQ](https://keda.sh/docs/latest/faq/)
+  - [FAQ](https://keda.sh/docs/2.13/reference/faq/)
```
Comment (chalin) [inline analyses/0011-keda/keda-implementation.md:140]: Thanks, this is indeed a better way to fix the link.

**Review by chalin — APPROVED** [review]
Did a quick read. Overall LGTM, thanks ✨

**Pull request merged**